code/modules/mob/living/simple_animal/friendly/slimehelper.dm

```diff
@@ -47,3 +44,13 @@
 
 /mob/living/simple_animal/hostile/helperslime/New()
 	B = new/obj/item/weapon/storage/bag/slime
+
+/mob/living/simple_animal/hostile/helperslime/AttackingTarget()
+	for(var/mob/M in view(src))
```

Comment:
This was a bit of example code I offered as a possible reason why his slime wasn't attacking slime cores, the logic being that perhaps somewhere in simple_animal code it only checks a list of mobs in view against `CanAttack()`, rather than atoms in view. I told him to look through simple_animal code for calls to `CanAttack()` and see if that was the case.

It seems he misunderstood me and added the non-functional example code to his own.